mxcube3/routes/SampleChanger.py

```diff
@@ -122,22 +122,38 @@ def mount_sample(loc):
 def unmount_sample(loc):
     mxcube.sample_changer.unload(loc, wait=True)
     set_current_sample(None)
+    try:
+        mxcube.sample_changer.unload(loc, wait=True)
+        set_current_sample(None)
+    except Exception as ex:
+	    return 'Cannot load sample', 409, {'Content-Type': 'application/json',
```

Comment:
The formatting here seems a bit funny, or is it just me ?

Reply:
I do not understand your comment

Reply:
The {} are missaligned for one and the dictionary keys as well ..., at least that's what I see here ?